DECISION-MAKING.md

```diff
@@ -0,0 +1,84 @@
+# Decision-making rules
+
+## Introduction
+
+The Brain Imaging Data Structure (BIDS) community set out the following 
+decision-making rules with the intention to:
+
+- Strive for consensus.
+- Promote open discussions.
+- Minimize the administrative burden.
+- Provide a path for when consensus cannot be made.
+- Grow the community.
+- Maximize the [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of the 
+  project.
+
+The rules outlined below are inspired by the [lazy consensus system used in the Apache Foundation](https://www.apache.org/foundation/voting.html) 
+and heavily depends on [GitHub Pull Request Review system](https://help.github.com/articles/about-pull-requests/).
+
+## Definitions
+
+**Repository** - [https://github.com/bids-standard/bids-specification](https://github.com/bids-standard/bids-specification)
+
+**Contributor** - a person listed in the Appendix I: Contributors. The 
+community decides on the content of this file using the same process as any 
+other change to the Repository (see below) allowing the meaning of "Contributor"
+to evolve independently of the Decision-making rules.
+
+## Rules
+
+1. Every modification of the specification (including a correction of a typo, 
+   adding a new Contributor, an extension adding support for a new data type, or
+   others) or proposal to release a new version needs to be done via a Pull 
+   Request (PR) to the Repository.
+1. Anyone can open a PR (this action is not limited to Contributors).
+1. A PR is eligible to be merged if and only if these conditions are met:
```

Comment:
This is an orthogonal comment to the ongoing discussion, but something like this could be incorporated in the [pull request template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) with checkboxes to tick before merging